packages/app-admin-ui/client/pages/Home/index.js

```diff
@@ -51,33 +52,11 @@ const Homepage = () => {
     },
   });
 
-  let allowedLists = lists;
-
   if (error) {
-    if (!error.graphQLErrors || !error.graphQLErrors.length) {
-      return (
-        <PageError>
-          <p>{error.message}</p>
-        </PageError>
-      );
-    }
-
-    const deniedQueries = error.graphQLErrors
-      .filter(({ name }) => name === 'AccessDeniedError')
-      .map(({ path }) => path && path[0]);
-
-    if (deniedQueries.length !== error.graphQLErrors.length) {
-      // There were more than Access Denied Errors, so throw a normal
-      // error message
-      return (
-        <PageError>
-          <p>{error.message}</p>
-        </PageError>
-      );
-    }
-
-    allowedLists = allowedLists.filter(
-      list => deniedQueries.indexOf(list.gqlNames.listQueryMetaName) === -1
+    return (
```

Comment:
We're no longer using Access Denied Errors to filter dashboard cards to show, so I figured it's better to just show any error.